src/main/java/com/ericsson/eiffel/remrem/semantics/EiffelEventType.java

```diff
@@ -62,6 +62,14 @@ public static EiffelEventType fromString(String eventType) {
         return eventTypeMap.get(eventType);
     }
 
+    public static EiffelEventType fromStringIgnoreCase(String eventType) {
+        if (eventTypeMap.size() == 0) {
```

Comment:
did not really get the point of using same map
twice, because it fails

Reply:
If we pass the eventtype as lowercase from cli or service, it will convert tolowercase from  Ignorecase method and check the ENUM valid or not.